packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -601,9 +601,9 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
             }
         });
         commandRegistry.registerCommand(CommonCommands.TOGGLE_MAXIMIZED, {
-            isEnabled: () => this.shell.canToggleMaximized(),
-            isVisible: () => this.shell.canToggleMaximized(),
-            execute: () => this.shell.toggleMaximized()
+            isEnabled: (event?: Event) => this.canToggleMaximized(event),
```

Comment:
Any existing command (that contributes context-menu items) such as the following will now need to pass `event` to work correctly?